CONTRIBUTING.md

```diff
@@ -1,12 +1,100 @@
-# Contributing
+# Contributing to UMF (Unified Memory Framework)
+
+- [Opening new issues](#opening-new-issues)
+- [Submitting Pull Requests](#submitting-pull-requests)
+    - [Building and testing](#building-and-testing)
+    - [Code style](#code-style)
+    <!-- - [Naming convention](#naming-convention) -->
```

Comment:
Either remove this or move to the top of this list.

Reply:
why? it doesn't break the list and when we add this section it's going to be right about here 😉